website/docs/tutorials/09-profiling.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_label: Profiling
+---
+
+# Profiling
+
+## Prerequisites
+
+You need pprof installed beforehand.
+You can learn how to install it [in pprof documentation](https://github.com/google/pprof#building-pprof).
+
+## How to profile a contract?
+
+Protostar can help you diagnose which functions are the most expensive in terms of used resources.
+Protostar currently measures:
+- number of steps
+- memory holes
+
+If you want to generate profile for a test case, run:
```

Comment:
```suggestion
If you want to generate a profile for a test case, run:
```
What is a profile?

Reply:
I think it is a common term, I don't think we should explain it. The same way we do not explain what terminal is.

Reply:
I think it is obvious from context what profile is. We have a sentence 
> Protostar can help you diagnose which functions are the most expensive in terms of used resources.

> A statistical summary of the events observed.

I do not think this is the meaning of word statistical. 

We could rephrase it to 
>Protostar can perform dynamic program analysis to help you diagnose which functions are the most expensive in terms of used resources.
 
but I am not sure if it is not unnecessary wordy. @mkaput WDYT?